README.md

```diff
@@ -53,6 +53,16 @@ dxscanner ~/my-project
 ```
 dxscanner [path]
```

Comment:
Set `dx-scanner` as default command to be consistent

Reply:
I was thinking about this actually :D Will do it :)